jmxtrans-utils/src/main/java/com/googlecode/jmxtrans/util/NumberUtils.java

```diff
@@ -43,4 +43,26 @@ public static boolean isNumeric(Object value) {
 		return false;
 	}
 
+	public static boolean isValidNumber(Object value) {
+		if (!(value instanceof Number)) {
```

Comment:
A number represented as a `String` is still treated as a number for most OutputWriters (including Graphite). We are probably missing a test case about this.